Sources/JWTKit/Signing/HMAC/HMACSigner.swift

```diff
@@ -1,35 +1,14 @@
-import CJWTKitBoringSSL
+import Crypto
+import Foundation
 
-internal struct HMACSigner: JWTAlgorithm {
-    let key: [UInt8]
-    let algorithm: OpaquePointer
+internal struct HMACSigner<SHA_TYPE>: JWTAlgorithm where SHA_TYPE: HashFunction {
```

Comment:
Upper snake case is a little weird here. Could this be `SHAType` or `Hash`?

Reply:
Fixed